### DISCUSSION TITLE
Video scaling settings for CDG

### DISCUSSION THREAD

**Discussion by rraszews** [vicwomg/pikaraoke#231] [Q&A]
I had a fairly easy time getting pikarioke up and running, and I'm loving how friendly it is to use, but I've got a bit of an annoyance since I'm working from a substantial collection of MP3+G files. Basically, if I plug my pi into a modern monitor, it looks... Pretty bad. The graphics are blurry and the color change is smeared heavily. I'm pretty sure this is down to the way the GPU is trying to scale the 320x216 CDG graphics to a modern HDMI resolution. 

I know that retrogamers using something like RetroPie are able to pick and choose scaling modes to make low-resolution video games look "right" on a modern screen. Does Pikaraoke have anything like that I can use to adjust the way it zooms the video?

**Comment by jramboz** [comment]
I believe pikaraoke uses vlc via command line to play mp3+cdg. Try looking at vlc's options to see what (if any) scaling options it provides. I'll do the same when I get a chance.

**Reply by rraszews** [reply]
That's a great point. I've been fiddling around with the various swscale options and the options for the openmax driver, but so far, no luck. I was able to get good results on an x86, so I think it's a matter of figuring out the right driver-specific options
